packages/core/src/symbol/SymbolUtil.ts

```diff
@@ -1337,6 +1337,8 @@ export class SymbolQuery {
 			}
 			this.#symbol = result
 			this.#map = result.parentMap
+			this.#parentSymbol = result.parentSymbol
+			this.path = result.path
```

Comment:
Since aliases are no longer used in this specific case, are these changes still needed? Or was this an issue that always existed before that you discovered while trying aliases?

Reply:
It's an issue that always existed